src/common/persistence/timescaledb/entities/sum-weekly.entity.ts

```diff
@@ -0,0 +1,33 @@
+import { PrimaryColumn, ViewColumn, ViewEntity } from 'typeorm';
+
+@ViewEntity({
+  expression: `
+    SELECT
+      time_bucket('7 day', timestamp) AS time, series, key,
+      last(value, timestamp) AS last,sum(value) AS sum
```

Comment:
same here

Reply:
I'll do this later on, with the next prs